modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc

```diff
@@ -74,18 +76,87 @@ Currently, not all tests run selectively on the nodes belonging to the pool.
 
 [id="cnf-performing-end-to-end-tests-running-the-tests_{context}"]
 == Running the tests
-Assuming the file is in the current folder, the command for running the test suite is:
+Assuming the kubeconfig file is in the current folder, the command for running the test suite is:
 
 ----
 $ docker run -v $(pwd)/:/kubeconfig -e KUBECONFIG=/kubeconfig/kubeconfig registry.redhat.io/openshift4/cnf-tests-rhel8:v4.6 /usr/bin/test-run.sh
 ----
 
 This allows your kubeconfig file to be consumed from inside the running container.
 
+[id="cnf-performing-end-to-end-tests-running-the-latency_tests_{context}"]
+=== Running the latency tests
+In OpenShift Container Platform 4.7, you can also run latency tests from the CNF-test container. The latency test allows you to set a latency limit, so that you can determine performance, throughput, and latency.
+
+The latency test runs the `oslat` tool, which is an open source program to detect OS level latency. For more information, see the Red Hat Knowledgebase solution link:https://access.redhat.com/solutions/5315541[How to measure OS and hardware latency on isolated CPUs?].
```

Comment:
Just FYI, It's okay to include this external link here even though we could not include an internal xref earlier in the document.